models/stg_facebook_pages__daily_page_metrics_total.sql

```diff
@@ -35,7 +35,7 @@ final as (
         page_actions_post_reactions_like_total as actions_post_reactions_like_total,
         page_actions_post_reactions_love_total as actions_post_reactions_love_total,
         page_actions_post_reactions_sorry_total as actions_post_reactions_sorry_total,
-        page_actions_post_reactions_total as actions_post_reactions_total,
+        coalesce(page_actions_post_reactions_total,(page_actions_post_reactions_anger_total + page_actions_post_reactions_haha_total + page_actions_post_reactions_like_total + page_actions_post_reactions_love_total + page_actions_post_reactions_sorry_total + page_actions_post_reactions_wow_total)) as actions_post_reactions_total,
         page_actions_post_reactions_wow_total as actions_post_reactions_wow_total,
```

Comment:
Very minor suggestion, I think it would be stylistically better to include all the reactions prior to the total calculation. This suggestion just moves the wow_total to above the reactions_total logic.
```suggestion
        page_actions_post_reactions_wow_total as actions_post_reactions_wow_total,
        coalesce(page_actions_post_reactions_total,(page_actions_post_reactions_anger_total + page_actions_post_reactions_haha_total + page_actions_post_reactions_like_total + page_actions_post_reactions_love_total + page_actions_post_reactions_sorry_total + page_actions_post_reactions_wow_total)) as actions_post_reactions_total,
```